pkg/shell/cockpit-dashboard.js

```diff
@@ -410,7 +410,18 @@ PageDashboard.prototype = {
                 update_series();
             }
 
-            return render;
+            function throttled_render() {
+                var timer = null;
+                return function () {
+                    if (timer)
+                        window.clearTimeout(timer);
```

Comment:
Better. If no timer -> render and set a timer, if there's a timer -> set a flag, if timer expires with flag set, re-render.